hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -1947,7 +1957,7 @@ protected int getDirectories(
             iterator = dirTable.iterator();
 
     iterator.seek(seekDirInDB);
-
+    countEntries = 0;
```

Comment:
If this is correct, you can make `countEntries` a local variable and remove it from the method parameter.

Reply:
+1